index.js

```diff
@@ -202,9 +205,7 @@ function startSimulatorProcess(options) {
 }
 
 
-function waitUntilSimulatorIsReady(port) {
-
-  var maxTimeout = 25000;
+function waitUntilSimulatorIsReady(port, maxTimeout) {
```

Comment:
Can this be an options hash, so we don't have to remember order of arguments? That'd be nicer ^_^